script.logviewer/resources/lib/navigation.py

```diff
@@ -12,6 +12,11 @@ def has_addon(addon_id):
     return xbmc.getCondVisibility("System.HasAddon({})".format(addon_id)) == 1
 
 
+def test_exception():
```

Comment:
leftover?

Reply:
It was left on purpose, so I don't have to write it every time I want to manually test an exception.